### ISSUE TITLE
Why filters are always hidden?

### ISSUE THREAD

**Issue by dkorsak** [sonata-project/SonataAdminBundle#2668]
$filer->isActive alawys returns false if filter does not has value. Empty filter does not has value by default

**Comment by pulzarraider** [comment]
In `master` version are filters hidden by default to make the [UI better](https://thomas.rabaix.net/talks/2014/sfpot-sonata-presentation/images/filters.gif).

Do you need the filters to be always available in your use case?

**Comment by webdevilopers** [comment]
In my use case a full text filter should permanently be visible for quick access even if the filter has not been used yet.

I tried a workaround by pre-setting filters but they did not have any effect:
```php
    public function getFilterParameters()
    {
        // Permanently display full text filter
        $this->datagridValues = array_merge(array(
                'full_text' => array (
                    'type'  => EqualType::TYPE_IS_EQUAL,
                    'value' => ' '
                )
            ), $this->datagridValues);
        return parent::getFilterParameters();
    }
```

Maybe we should add an filter parameter `filter_hidden` that can be set to `false`?

**Comment by pulzarraider** [comment]
@dkorsak @webdevilopers fixed with PR #2751 (option `'show_filter'=>true` is here to make the filter always visible). Thank you for reporting the issue.
